socorro/external/es/supersearch.py

```diff
@@ -511,7 +511,11 @@ def get(self, **kwargs):
                         # The reason cannot be found in the exception info, or
                         # we cannot find anything while parsing the exception.
                         pass
-                raise
+                # If we don't raise the original exception explictly we run we
+                # would actually raise other (even swallowed) exceptions that
+                # might have happened during this block here were we peek
+                # into the exception object.
+                raise e
```

Comment:
This hoses the exception context.

A better way to deal with this is at the beginning of this exception handling block, you should capture `sys.exc_info()`. Then raise those things. For example, we do that here:

https://github.com/mozilla-services/socorro/blob/059d82c6f03603ac6ad39d014fbb99b85618820a/socorro/processor/processor_app.py#L144-L185

Pretty sure that handles your needs here while preserving the exception context correctly.

Reply:
Good catch! Thanks!